TrackletAlgorithm/AllStubMemory.h

```diff
@@ -163,6 +163,14 @@ class AllStub : public AllStubBase<ASType>
     return data_.range(kASBendMSB,kASBendLSB);
   }
 
+  bool isPSStub() const {
+    if(ASType == BARRELPS) return true;
+    else if(ASType == BARREL2S) return false;
+    else if(ASType == DISKPS || ASType == DISK2S) {
+      return data_.range(getWidth()-1, getWidth()-3) > 0; // Check highest 3 bits regardless of template type
```

Comment:
Can you expand the comment and say that this checks the radius and if these are zero it means that you have 2S stubs where the radius just encodes the ring number and is less than 16.

Reply:
Done